tests/plugins/stylescombo/manual/stylewithnoselection.html

```diff
@@ -0,0 +1,28 @@
+<textare id="editor">
+	Hello World
+</textare>
+
+<script>
+	if ( bender.tools.env.mobile ) {
```

Comment:
Error was thrown only in console. Generally during tests it's hard to analyse what happen on mobile console. So even if bug potentially might still exist, it won't be visible in such environment. That's why manual tests are ignored on mobile for this fix.